client/web/src/site-admin/WebhookCreateUpdatePage.tsx

```diff
@@ -333,8 +333,8 @@ function convertWebhookToCreateWebhookVariables(webhook: Webhook): CreateWebhook
     }
 }
 
-function codeHostSupportsSecretes(codeHostKind: ExternalServiceKind): boolean {
-    if (codeHostKind === ExternalServiceKind.BITBUCKETCLOUD || codeHostKind === ExternalServiceKind.AZUREDEVOPS) {
+function codeHostSupportsSecrets(codeHostKind: ExternalServiceKind): boolean {
+    if (codeHostKind === ExternalServiceKind.AZUREDEVOPS) {
```

Comment:
I wonder if would be better to use an exhaustive switch statement here (with no default)? This way the typescript compiler can help us in the future if we had a new code host kind by telling us when we haven't handled a case.

Ex 
```suggestion
    switch (codeHostKind) {
        case ExternalServiceKind.AZUREDEVOPS:
            return false
        case ExternalServiceKind.BITBUCKETCLOUD:
            return true
        // ...
    }
```